### PR TITLE
Updated Organization model and loader

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-5191 and ECOM-5192

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: course_discovery/apps/course_metadata/migrations/0013_auto_20160809_1259.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ class Migration(migrations.Migration):
             name='subject',
             unique_together=set([('partner', 'name'), ('partner', 'slug'), ('partner', 'uuid')]),
         ),
-        migrations.RunPython(update_subjects, lambda *args: None),
+        migrations.RunPython(update_subjects, reverse_code=migrations.RunPython.noop),
```
Comment (rlucioni) [inline course_discovery/apps/course_metadata/migrations/0013_auto_20160809_1259.py:85]: Nice.

File: course_discovery/apps/course_metadata/models.py [hunk 2]
```diff
@@ -152,18 +152,22 @@ class MajorWork(AbstractNamedModel):
 
 class Organization(TimeStampedModel):
     """ Organization model. """
-    key = models.CharField(max_length=255, unique=True)
-    name = models.CharField(max_length=255, null=True, blank=True)
+    partner = models.ForeignKey(Partner, null=True, blank=False)
+    uuid = models.UUIDField(blank=False, null=False, default=uuid4, editable=False, verbose_name=_('UUID'))
```
Comment (rlucioni) [inline course_discovery/apps/course_metadata/models.py:156]: Why is this field optional?

Comment (clintonb) [inline course_discovery/apps/course_metadata/models.py:156]: It should not be.

Comment (clintonb) [inline course_discovery/apps/course_metadata/models.py:156]: This field is not optional.

Comment (rlucioni) [inline course_discovery/apps/course_metadata/models.py:156]: I could swear I saw `blank=True` before. (`null` and `blank` default to `False`, so you could exclude them here.)

**Comment by rlucioni** [comment]
👍

**Pull request merged**